### PR TITLE
R blas

### PR DESCRIPTION
**Pull request by bnaras**
Adding a source of blas from R.

### CODE REVIEW

**Comment by bnaras** [comment]
@IvanUkhov thanks for your review: I've addressed those issues. 

I am not sure what the next step is. Should I merge this one and the lapack one too? However, the CI is failing and I saw that the jobs aren't picked by a runner.

**Comment by bnaras** [comment]
The CI is failing because both [macOS 10.15](https://github.blog/changelog/2022-07-20-github-actions-the-macos-10-15-actions-runner-image-is-being-deprecated-and-will-be-removed-by-8-30-22/)  and [Ubuntu 18.04](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/) have been deprecated. So no runners are available. I will try to update the workflow.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ The following implementations are available:
 * `intel-mkl`, which is the one in [Intel MKL],
 * `netlib`, which is the reference one by [Netlib], and
 * `openblas`, which is the one in [OpenBLAS].
+* `r`, which is the one in [R].
```
Comment (IvanUkhov) [inline README.md:16]: It was a bit off if you look at the whole list. The “and” should have been moved from line 14 to line 15.

File: README.md [hunk 2]
```diff
@@ -23,6 +24,7 @@ blas-src = { version = "0.8", features = ["blis"] }
 blas-src = { version = "0.8", features = ["intel-mkl"] }
 blas-src = { version = "0.8", features = ["netlib"] }
 blas-src = { version = "0.8", features = ["openblas"] }
+r-src = { version = "0.8", features = ["r"] }
```
Comment (IvanUkhov) [inline README.md:27]: Here it is also off. It should have been `blas-src = …`.

File: src/lib.rs [hunk 3]
```diff
@@ -11,6 +11,7 @@
 //! * `intel-mkl`, which is the one in [Intel MKL],
 //! * `netlib`, which is the reference one by [Netlib], and
 //! * `openblas`, which is the one in [OpenBLAS].
+//! * `r`, which is the one in [R].
```
Comment (IvanUkhov) [inline src/lib.rs:14]: The same here about the punctuation.

File: src/lib.rs [hunk 4]
```diff
@@ -21,6 +22,7 @@
 //! blas-src = { version = "0.8", features = ["intel-mkl"] }
 //! blas-src = { version = "0.8", features = ["netlib"] }
 //! blas-src = { version = "0.8", features = ["openblas"] }
+//! blas-src = { version = "0.8", features = ["r"] }
```
Comment (IvanUkhov) [inline src/lib.rs:25]: Here it is correct, unlike the readme.

**Comment by IvanUkhov** [comment]
Sorry for the slow response. I added more comments. Since this is merged, one would have to do it elsewhere. You can push directly to main if you would like, as it is minor changes.

**Comment by IvanUkhov** [comment]
And thank you for looking into the CI!

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -22,21 +22,17 @@ jobs:
...
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
       - name: Run the test suite
         uses: actions-rs/cargo@v1
         with:
           command: test
           args: >
             --features=${{ matrix.feature }}
-        env:
```
Comment (IvanUkhov) [inline .github/workflows/build.yml:32]: Can it be failing because you removed these variables?

**Pull request merged**